### ISSUE TITLE
[Accessibility] fd-form-message content not announced

### ISSUE THREAD

**Issue by m8524769** [SAP/fundamental-ngx#11595]
#### Is this a bug, enhancement, or feature request?

bug

#### Describe your proposal.

On accessing input fields with `fd-form-message` using keyboard/mouse, the message should be announced.

#### Can you handle that on the application side

no

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

0.43.21

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

Access input field with `fd-form-message`.

#### Did you check the documentation and the API?

yes

#### Did you search for similar issues?

yes

**Comment by droshev** [comment]
@m8524769  did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side?

**Comment by m8524769** [comment]
> @m8524769 did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side?

Hi @droshev, thanks for your reply, it could be reproduced in the latest [documentation](https://sap.github.io/fundamental-ngx/#/core/input#states) and I don't think we should set aria attributes on the application side.

**Comment by mikerodonnell89** [comment]
@m8524769 Because the `<fd-form-input-message-group>` component (like from the link you provided) is agnostic to whatever is placed inside of it via `ng-content`, we advise that you should handle aria attributes on the application side. If we were to set these attribrutes on the `fd-form-message` internally in the library to force an announcement, it would potentially override or interrupt whatever the screenreader had originally planned to say, possibly set by the application. So unfortunately it is best to handle this application side, like you see in the examples here: https://sap.github.io/fundamental-ngx/#/core/form-message

However, our documentation in the link you provided should be utilizing aria-describedby so I'll be updating that
